fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalWindowOperator.java

```diff
@@ -56,6 +56,9 @@ public class LogicalWindowOperator extends LogicalOperator {
     private boolean useHashBasedPartition;
     private boolean isSkewed;
 
+    // only true when local partition top N can pre agg
+    private boolean inputIsBinary;
```

Comment:
> Do we have to check on the FE side?

this is needed for plan fragment builder to tell analytic operator that it's analytic function should use merge instead of update:
```
if (node.isInputIsBinary()) {
                    ((FunctionCallExpr) analyticFunction).setMergeAggFn();
}
```
And to help Typechecker to skip check